doc/source/configuration/external_satpy.rst

```diff
@@ -23,32 +23,40 @@ It is your responsibility to make sure the setting points to a suitable Satpy
 package: If the given path doesn't point to a Python package directory or not to
 one providing Satpy, the application may exit immediately throwing Exceptions.
 
-Using Extra Readers
-===================
+Using Extra Satpy component configuration
+=========================================
+
+SIFT can use external `Satpy component configuration <https://satpy.readthedocs.io/en/stable/config.html#component-configuration>`_ folder,
+that hosts extra ``readers``, ``composites``, ``enhancements`` and ``areas`` definitions.
+To use external satpy component configuration it is necessary to define
+``satpy_extra_config_path`` in `external_satpy.yaml <https://github.com/ssec/sift/blob/master/uwsift/etc/SIFT/config/external_satpy.yaml>`_: ::
```

Comment:
```suggestion
``satpy_extra_config_path`` in `external_satpy.yaml <https://github.com/ssec/sift/blob/master/uwsift/etc/SIFT/config/external_satpy.yaml>`_::
```

Reply:
This should probably refer to where this file shows up in the final installation too, right?

Reply:
Sorry, I meant maybe you could also mention (in addition to linking to the github version of the YAML) that in the final installation (when someone installs SIFT) that this file would show up in `SIFT_directory/lib/python3.11/site-packages/uwsift/etc/...` or wherever it ends up.

Reply:
Actually, as described here https://sift.readthedocs.io/en/latest/configuration/index.html#configuration, the idea is that users do not modify the default configuration (e.g. in `SIFT_directory/lib/python3.11/site-packages/uwsift/etc/`), but rather place the personal settings in the user config path (e.g. `~/.config/SIFT/config/`), so that in the case of an update, the configuration modifications are not lost. So I would refrain in pointing out where the defaults are, if you agree. 

Also, technically the user config doesn't need to be in a file with a specific name; the file just needs to be a .yaml file inside the user config folder. So I would also not mention `external_satpy.yaml` in the documentation, but rather simply say `in the user configs`

Reply:
Ah right. Ok but will `external_satpy.yaml` show up in a users config directory by default? No, right? So this `external_satpy.yaml` is just an example. So this section of the docs should maybe point the user to their personal configuration directory and say that `external_satpy.yaml` is an example?

Reply:
No it won't, the folder is empty (apart from the empty subfolder `readers`). Yep, agree.

Reply:
No it won't, the folder is empty (apart from the empty subfolder `readers`). Yep, agree.

Reply:
Ok, I made f7287c405c5490d80b1abde391a9b46e1f4f02a4